cea/interfaces/dashboard/landing/routes.py

```diff
@@ -160,6 +176,41 @@ def route_open_project_scenario(scenario):
     return redirect(url_for('inputs_blueprint.route_table_get', db='zone'))
 
 
+@blueprint.route('/get-image/<scenario>')
+def route_get_images(scenario):
+    cea_config = current_app.cea_config
+    project_path = cea_config.project
+    zone_path = os.path.join(project_path, scenario, 'inputs', 'building-geometry', 'zone.shp')
```

Comment:
here again - let's use the locator to get hold of these files. instead of assuming they're inside a specific folder structure. that's what the `InputLocator` does: it decouples the storage location (folder structure) from all the scripts and makes it a single point to change, if necessary. also, the next step is to decouply storage _type_ (files vs. mysql or whatever).